tigerpath/majors_and_certificates/majors/FIT.json

```diff
@@ -124,10 +123,27 @@
                   "max_counted": null,
                   "min_needed": 3,
                   "explanation": null,
-                  "course_list": [
-                    "FRE 3*",
-                    "FRE 4*",
-                    "FRE 5*"
+                  "req_list": [
+                    {
+                      "name": "Junior Seminar",

```

Comment:
https://ua.princeton.edu/academic-units/department-french-and-italian

For concentrators in French in the classes of 2021 and beyond, the Junior Seminar (FRE 398), is mandatory.